README.md

```diff
@@ -50,6 +50,14 @@ The following are required and recommended tools for using this cookiecutter and
     ```
     You may also choose to not have a persistent installation of cruft, in which case you would replace any calls to the `cruft` command below with `pipx run cruft`. 
 
+
+  * **make or just as command runner**
+
+    The project contains a makefile but also a `justfile` with pre-defined complex commands. To execute these commands you either need `make` or [just](https://github.com/casey/just) as an alternative command runner. Especially for Windows users we suggest `just`. Install it by running:
```

Comment:
I'm a little concerned about having two command runners requiring changes in two places and the possibility of their getting out of sync. Do you have a strategy for managing both of these files to avoid that?

Reply:
Yes. I agree that having both is not ideal but maybe worth an experiment. Some make users may also convert to just but it is difficult to predict.